### PR TITLE
Azure Cloud Adoption Framework with Datadog Guide

### PR DESCRIPTION
**Pull request by estib**
### What does this PR do?
Adds a guide for how to use Datadog to measure the success of an Azure cloud migration for orgs following Azure's Cloud Adoption Framework. This guide is part of a new partnership with Azure's Cloud Adoption Framework (CAF) team, which will be released in mid-February

### Motivation
As of mid-February, Datadog will be among the official partners of the Cloud Adoption Framework. CAF users coming to Datadog through this partnership will need a guide to help them use Datadog in the context of the CAF method to migrate safely to Azure cloud. 

### Preview

https://docs-staging.datadoghq.com/estib/azure-cloud-adoption-framework/integrations/guide/azure-cloud-adoption-framework/
https://docs-staging.datadoghq.com/estib/azure-cloud-adoption-framework/integrations/guide/

### Additional Notes

This guide will likely change a lot in the future as we automate more of the onboarding process for CAF customers. For now it's really just a "welcome to Datadog" guide with a very specific customer audience in mind. 

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/integrations/guide/azure-cloud-adoption-framework.md [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+1. In your Datadog account, [create a new dashboard][22].
+2. Select the 'cog' at the top right corner
+3. Select 'import dashboard content' ([setting documented here][23])
+4. Paste in the JSON definition of the dashboard found [here][24]
+
+## Send Actionable Alerts to Workload Owners
+
+As you migrate your workloads, you will want to make sure the right people can be automatically alerted when important performance KPIs thresholds are crossed. To do this, create monitors in Datadog that constantly observe the health of your workloads and trigger notifications to your communication channels, wether over MS Teams, Slack, paging services, or ticketing systems.
+
+We recommend reviewing [this guide on how to set up actionable and effective alerts][25]. If your tags map effectively to dedicated investigative channels (e.g, if you have a Teams channel for every "owner" or "team" or "workspace", then you can configure your monitors to [dynamically notify the appropriate communication channel][10] with monitor template variables.
```
Comment (alai97) [inline content/en/integrations/guide/azure-cloud-adoption-framework.md:102]: Is this guide link to /monitors?

```suggestion
We recommend reviewing [this guide on how to set up actionable and effective alerts][25]. If your tags map effectively to dedicated investigative channels (e.g, if you have a Teams channel for every "owner" or "team" or "workspace", then you can configure your monitors to [dynamically notify the appropriate communication channel][10] with monitor template variables. 
```

**Review by alai97 — COMMENTED** [review]
Some questions before submitting my doc review!

File: content/en/integrations/guide/azure-cloud-adoption-framework.md [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+1. Enable Datadog-Azure integrations that your new workloads will use to indicate their performance and health
+2. Document a tagging strategy that your teams will use to describe their workloads as they migrate
+3. Configure dashboards that stakeholders can use to follow migration progress and understand the overall health of the new workloads
+4. Establish communication channels for incident response
+
+## Enable Datadog-Azure Integrations
+
+If you don't yet have a Datadog account, you can [start a free 2-week trial from this link][1].
+
+Datadog and Azure partner together to offer Datadog's services from within your Azure account. For each of your landing zones, you can create a Datadog Resource to link your Datadog account to your Azure account and access your observability data from within Azure. [This guide][2]will walk you through that process, and will help you determine which of your Azure data to collect into Datadog.
```
[the remark this answers is not in the document]
Comment (estib) [reply content/en/integrations/guide/azure-cloud-adoption-framework.md]: thanks for asking -- the link for that guide should be this `/integrations/azure/?tab=link&site=us3`

File: content/en/integrations/guide/azure-cloud-adoption-framework.md (hunk 1 above)
Comment (estib) [reply content/en/integrations/guide/azure-cloud-adoption-framework.md]: Yep, just `/monitors/`

File: content/en/integrations/guide/azure-cloud-adoption-framework.md [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+Migrating your services from on-premises or other cloud environments to a new cloud environment can be risky. 
+
+Azure's Cloud Adoption Framework has a proven strategy for successful migrations and Datadog's monitoring ensures safe, speedy migrations by measuring the health of your new and original environments in the same system of observability and incident response. 
+
+You can:
+
+1. Prepare your Datadog account for your teams to efficiently execute their workload migrations. This is the "planning" stage.
+2. Use Datadog to measure the health of your original environments and new workloads as your teams launch them in your new landing zones. This is the "migrating" stage. 
+
+This guide documents the migration process for organizations that follow Azure's Cloud Adoption Framework.
```
Comment (alai97) [inline content/en/integrations/guide/azure-cloud-adoption-framework.md:21]:
```suggestion
This guide documents the migration process for organizations that follow Azure's Cloud Adoption Framework.

If you don't have a Datadog account yet, you can [start a two-week trial][2].
```

File: content/en/integrations/guide/azure-cloud-adoption-framework.md [hunk 4]
```diff
@@ -0,0 +1,184 @@
...
+1. Enable the [Datadog-Azure integration][1] so your new workloads can indicate their performance and health.
+2. Document a tagging strategy that your teams can use to describe their workloads as they migrate.
+3. Configure dashboards that stakeholders can use to follow along the migration progress and understand the overall health of the new workloads.
+4. Establish communication channels for incident response.
+
+### Enable Datadog-Azure Integrations
+
+Datadog and Azure have partnered together to offer Datadog's services within your Azure account. For each one of your landing zones, you can create a Datadog resource to link your Datadog account with your Azure account and access your observability data in Azure. 
+
+If you don't have a Datadog account yet, [start a two-week trial][2]. For more information about this process, see the [Datadog-Azure Integration][1].
```
Comment (alai97) [inline content/en/integrations/guide/azure-cloud-adoption-framework.md:36]:
```suggestion
For more information about this process to help you determine what Azure data you want to collect in Datadog, see the [Microsoft Azure documentation][1].
```

**Review by alai97 — COMMENTED** [review]
Two copy nits!

**Review by alai97 — APPROVED** [review]

**Pull request merged**